ci/sarusify.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Get the full paths of the test binaries
+TEST_BINARIES=`ctest -L MPI -V -N | grep -oP "(?<=\")(/root/DLA-Future-build.*?)(?=\")"`
+
+# Copy all test commands over to a single shell script
+SARUS_TEST_COMMANDS=$(ctest -L MPI -V -N | sed -n -e 's/^.*Test command: //p' | sed 's|/usr/bin/srun|srun|g')
```

Comment:
Non MPI tests are not executed...
Can the command `srun -n 1 -c 36 sarus run ctest -L NOMPI` be added?